FosServer/src/main/java/com/example/fostracker/models/Name.java

```diff
@@ -0,0 +1,38 @@
+package com.example.fostracker.models;
+
+/** Serves as a model for names so that Store and Agent classes are simplified and redundancy is avoided */
```

Comment:
This is a good idea, commonly used classes such as name, phone number should be reused.